nms_stack/nms_cli/nms_stack/group_vars/all

```diff
@@ -123,7 +123,7 @@ docker_internal_network: fd00:420:69::/48
 swarm_manager_port: 2377
 
 # TG Docker Registry
-docker_registry_url: https://secure.cxl-terragraph.com:443/v2
+docker_registry_url: https://ghcr.io/terragraph/tgnms
```

Comment:
should this be ghcr.io/terragraph or ghcr.io/terragraph/tgnms ?

Reply:
fixed it.